launchable/commands/subset.py

```diff
@@ -526,88 +534,111 @@ def get_payload(
 
             return payload
 
+        def _collect_test_like_files(self):
```

Comment:
`test_like` is slightly ambiguous for me. How about `_collect_candidate_test_files` or `_collect_potential_test_files`?